src/internal/kopia/upload.go

```diff
@@ -1185,20 +1178,32 @@ const (
 func inflateBaseTree(
 	ctx context.Context,
 	loader snapshotLoader,
-	snap ManifestEntry,
+	base BackupBase,
 	updatedPaths map[string]path.Path,
 	roots map[string]*treeMap,
 ) error {
+	bupID := "no_backup_id"
```

Comment:
I understand this would only stay set in some weird pathological case, but does this have any significance?